gen2-triangulation/README.md

```diff
@@ -0,0 +1,39 @@
+# Gen2 Triangulation 3D visualizer
+
+This is a Gen2 equivalent of `triangulation-3D-visualizer`.
+Because there are often application-specific host-side filtering to be done on the stereo
+neural inference results, and because these calculations are lightweight
+(i.e. could be done on an ESP32), we leave the triangulation itself to the host.
+
+This 3D visualizer is for the facial landmarks demo, and uses OpenGL and OpenCV.
+Consider it a draft/reference at this point.
+
+## Demo
+
+[![Spatial AI](https://user-images.githubusercontent.com/5244214/90748450-7c9e9a80-e2d2-11ea-9e9e-da65b5d9e6f0.gif)](https://www.youtube.com/watch?v=Cs8xo3mPBMg "3D Facial Landmark visualization")
+
+Note that this is Gen1 video demo
```

Comment:
I think we should record a new video instead

Reply:
added now